packages/debugger-extension/src/index.ts

```diff
@@ -721,10 +721,11 @@ const main: JupyterFrontEndPlugin<void> = {
     });
 
     commands.addCommand(CommandIDs.pause, {
-      label: service.isPausingOnExceptions
-        ? trans.__('Disable pausing on exceptions')
-        : trans.__('Enable pausing on exceptions'),
-      caption: trans.__('Enable / Disable pausing on exceptions'),
+      label: trans.__('Pause on exceptions'),
```

Comment:
For consistency with other items in palette we may prefer:

```suggestion
      label: trans.__('Pause on Exceptions'),
```

but this is not a strong opinion.